consent-ws/src/main/java/org/broadinstitute/consent/http/service/DataSetFileParser.java

```diff
@@ -14,13 +15,18 @@
 
 public class DataSetFileParser {
 
-    private final String BLANK_REQUIRED_FIELD = "Dataset ID %s - The required field: %s is empty in row %d.";
+    private final String BLANK_REQUIRED_FIELD = "Dataset %s - The required field: %s is empty in row %d.";
+    private final String BLANK_REQUIRED_FIELDS = "Dataset %s - Consent ID or Sample Collection ID is required";
     private final String MISSING_COLUMNS = "Your file has more/less columns than expected. Expected quantity: %s";
     private final String MISSING_MISPLACED_HEADER = "The uploaded file does not comply with the accepted fields. Field: (%s)%s is not recognized/ordered correctly. It should be '%s'";
     private final String PLEASE_DOWNLOAD = "Please download the Dataset Spreadsheet Model from the 'Add Datasets' window.";
-    private int DATASET_ID_INDEX = 9;
+    private int DATASET_NAME_INDEX = 0;
+    private int SAMPLE_COLLECTION_INDEX = 9;
+    private int CONSENT_ID_INDEX = 10;
+    private String SECOND_DATASET_NAME = "Melanoma_Regev";
+    private String FIRST_DATASET_NAME = "Melanoma-Regev-Izar-Garraway-DFCI-ICR";
```

Comment:
See other comments about hard-coding dataset names. I think this could be less brittle with either a configuration or a new field on the dataset table.